osmnx/utils.py

```diff
@@ -573,6 +618,125 @@ def get_nearest_nodes(G, X, Y, method=None):
     return np.array(nn)
 
 
+def get_nearest_edges(G, X, Y, method=None, dist=10):
+    """
+    Return the graph edges nearest to a list of points. Pass in points
+    as separate vectors of X and Y coordinates. The 'kdtree' method
+    is by far the fastest with large data sets, but only finds approximate
+    nearest edges if working in unprojected coordinates like lat-lng (it
+    precisely finds the nearest edge if working in projected coordinates).
+
+    The method creates equally distanced points along the edges of the network.
+    Then, these points are used in a kdTree search to find the nearest point along
+    an edge.
+
+    This method will not give the exact perpendicular point along the edge, but the
+    smaller the *dist* parameter, closer the solution will be.
+
+    Code is adapted from an answer by JHuw from this original question:
+    https://gis.stackexchange.com/questions/222315/geopandas-find-nearest-point-in-other-dataframe
+
+    Parameters
+    ----------
+    G : networkx multidigraph
+    X : list-like
+        The vector of longitudes or x's for which we will find the nearest
+        edge in the graph
+    Y : list-like
+        The vector of latitudes or y's for which we will find the nearest
+        edge in the graph
+    method : str {None, 'kdtree'}
+        Which method to use for finding nearest edge to each point.
+        If None, we manually find each edge one at a time using
+        osmnx.utils.get_nearest_edge. If 'kdtree' we use
+        scipy.spatial.cKDTree for very fast euclidean search.
+    dist : float
+        spacing  distance for the point creation along edges. The smaller, the more points are created.
```

Comment:
Are these in the same units as the graph? That is, by default they are in degrees? And for projected graphs, in meters? Would be useful to explain here in the docstring.

Reply:
Yes that's right.